client/my-sites/patterns/components/get-access-modal/index.tsx

```diff
@@ -22,7 +27,10 @@ export const PatternsGetAccessModal = ( { isOpen, onClose }: PatternsGetAccessMo
 			isVisible={ isOpen }
 			additionalClassNames="patterns-get-access-modal"
 			additionalOverlayClassNames="patterns-get-access-modal__backdrop"
-			onClose={ onClose }
+			onClose={ () => {
+				onClose();
+				tracksEventHandler( 'calypso_pattern_library_get_access_dismiss' );
+			} }
 		>
 			<div className="patterns-get-access-modal__content">
 				<button className="patterns-get-access-modal__close" onClick={ onClose }>
```

Comment:
Forgot to handle also `X` click:
```suggestion
				<button
					className="patterns-get-access-modal__close"
					onClick={ () => {
						onClose();
						tracksEventHandler( 'calypso_pattern_library_get_access_dismiss' );
					}}
				>
```